src/hooks/useContentView.js

```diff
@@ -0,0 +1,21 @@
+import { useContext, useEffect } from 'react';
+import { ContentViewContext } from 'contexts/ContentViewContext';
+
+import Components from 'components/Constants/Components';
+
+const useContentView = () => {
```

Comment:
```suggestion
export const useContentView = () => {
```
check other cases, do you remember the named exports comments in another PR?